tests/Happyr/LinkedIn/LinkedInTest.php

```diff
@@ -35,6 +36,19 @@ public function testConstructor()
             'Expect the API secret to be set.');
     }
 
+    public function testInitProperlyAssignsPassedInStorage()
```

Comment:
This test does not do much.. Try change the init function and you will see that this test still will pass. Consider changing it to: 

```php
        $sessionStorage = new IlluminateSessionStorage();
        $ln = new LinkedIn(self::APP_ID, self::APP_SECRET, $sessionStorage);
        $this->assertInstanceOf('Happyr\LinkedIn\Storage\IlluminateSessionStorage', $ln->getStorage());
```